contrib/hyperscan/matching/input_matchers/test/config_test.cc

```diff
@@ -45,12 +39,12 @@ class ConfigTest : public ::testing::Test {
     for (auto& setup_config : setup_configs) {
       std::string option_strs;
       for (auto& option : setup_config.second) {
-        option_strs += fmt::format(optionString(), option.first, option.second);
+        option_strs += fmt::format(option_string.data(), option.first, option.second);
```

Comment:
never take data() from a string_view (or string) unless you are also taking size -- there's no null termination guarantee.

Moreover, why do you need that? Doesn't fmt::format take a string_view directly? If not I guess you need to use std::string(options_string).

Reply:
No, `fmt::fotmat` cannot take a `absl::string_view` directly. I will take your suggestion to use `std::string(absl::string_view)`.